packages/components/dialog/src/dialog.scss

```diff
@@ -38,6 +41,7 @@ dialog {
     100vh - var(--_margin-top-container) - var(--_margin-bottom-container) - 2 * var(--_padding-block)
   );
   max-inline-size: var(--_max-inline-size);
+  min-inline-size: 100dvw;
```

Comment:
I think that we wanted to have a different inline-size before? :) 
https://www.figma.com/design/1oIVcD9VMbOtLXYp48ehsA/Sanoma-Learning-Light-mode?node-id=5303-20817&t=5rOdHrMaGkLLoMTe-4

@arecuenco-dsgn do we want to change it this way?

But maybe this is sth already discussed :)

Reply:
There is a bug in any case here on mobile:
![CleanShot 2024-07-04 at 15 58 18@2x](https://github.com/sl-design-system/components/assets/3968/4872063a-a5b2-4e3f-a3ac-4faead13820c)

The `100dvw` change fixes that.